pkg/querier/queryrange/roundtrip_test.go

```diff
@@ -24,14 +24,14 @@ import (
 	"github.com/grafana/loki/pkg/logproto"
 	"github.com/grafana/loki/pkg/logqlmodel"
 	"github.com/grafana/loki/pkg/querier/queryrange/queryrangebase"
-	"github.com/grafana/loki/pkg/storage/chunk"
 	"github.com/grafana/loki/pkg/storage/chunk/cache"
+	"github.com/grafana/loki/pkg/storage/config"
 	util_log "github.com/grafana/loki/pkg/util/log"
 	"github.com/grafana/loki/pkg/util/marshal"
 )
 
 var (
-	testTime   = time.Date(2019, 12, 02, 11, 10, 10, 10, time.UTC)
+	testTime   = time.Date(2019, 12, 0o2, 11, 10, 10, 10, time.UTC)
```

Comment:
```suggestion
	testTime   = time.Date(2019, 12, 02, 11, 10, 10, 10, time.UTC)
```
Guessing you didn't mean to use an octal literal here, even though it's the same :)

Reply:
LOL